src/Microsoft.SqlTools.ServiceLayer/Utility/SqlScriptFormatters/Common.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.SqlTools.ServiceLayer.Utility.SqlScriptFormatters
+{
+    public class Common
```

Comment:
Can you remove this since it's not implementing anything at all?

Reply:
It *was*... I guess it isn't anymore